Gemfile.lock

```diff
@@ -63,19 +63,14 @@ GEM
     rb-fsevent (0.10.4)
     rb-inotify (0.10.1)
       ffi (~> 1.0)
-    rspec (3.9.0)
-      rspec-core (~> 3.9.0)
-      rspec-expectations (~> 3.9.0)
-      rspec-mocks (~> 3.9.0)
-    rspec-core (3.9.3)
-      rspec-support (~> 3.9.3)
-    rspec-expectations (3.9.2)
-      diff-lcs (>= 1.2.0, < 2.0)
-      rspec-support (~> 3.9.0)
-    rspec-mocks (3.9.1)
-      diff-lcs (>= 1.2.0, < 2.0)
-      rspec-support (~> 3.9.0)
-    rspec-support (3.9.3)
+    rspec (2.99.0)
```

Comment:
Not necessarily a blocking comment, but why did you need to downgrade RSpec here?

I can see before this commit you didn't add a test the failed due to RSpec - was something failing on master?

Reply:
It was failing on master because some configuration was not working. I decided to downgrade the gem instead of adjusting the suite which I'm not sure if works.